perfkitbenchmarker/providers/alicloud/util.py

```diff
@@ -72,13 +72,12 @@ def AddTags(resource_id, resource_type, region, **kwargs):
   tag_cmd = ALI_PREFIX + [
       'ecs',
       'AddTags',
-      '--RegionId %s' % region,
-      '--ResourceId %s' % resource_id,
-      '--ResourceType %s' % resource_type]
-  for index, (key, value) in enumerate(six.iteritems(kwargs)):
-    tag_cmd.append('--Tag{0}Key {1} --Tag{0}Value {2}'
-                   .format(index + 1, key, value))
-  tag_cmd = GetEncodedCmd(tag_cmd)
+      '--RegionId', region,
```

Comment:
Is the space no longer required?

Reply:
This is related to the `GetEncodedCmd` explanation below. The current version already provides the arguments in shell-like syntax (flag names and arguments as separate list items), so a space here is no longer necessary.

Reply:
Ok.